### PR TITLE
Update gems to remove warnings

### PR DESCRIPTION
**Pull request by DavidKang**

### CODE REVIEW

File: src/api/spec/nyan_unicorn_formatter.rb [hunk 1]
```diff
@@ -1,6 +1,29 @@
 # -*- coding: utf-8 -*-
 require 'nyan_cat_formatter'
 
+class RSpec3
```
Comment (hennevogel) [inline src/api/spec/nyan_unicorn_formatter.rb]: Please at least explain what this does in a comment....

Comment (DavidKang) [inline src/api/spec/nyan_unicorn_formatter.rb]: I added this class to overwrite and fix the deprecation warning of nyan-cat-formatter, until the PR is accepted to them. Although the gem looks like abandoned.

**Comment by bgeuken** [comment]
LGTM

**Comment by Ana06** [comment]
[(5) Update GEMs to avoid deprecation warnings](https://trello.com/c/iqC2G6v8/207-5-update-gems-to-avoid-deprecation-warnings)

**Comment by mmohring** [comment]
Hi, could you also check the rake version ? When I run `RAILS_ENV=production rake db:migrate` I get the error that the rake version is wrong.

**Pull request merged**